syz-verifier/main.go

```diff
@@ -65,6 +65,7 @@ func main() {
 		if err != nil {
 			log.Fatalf("%v", err)
 		}
+		defer pi.pool.Close()
```

Comment:
Nit: I think a TODO would have made more sense in this case
1) The syz-verifier tool is currently completely broken and needs a big rewrite
2) This function never returns

Reply:
Ah, thanks, I wasn't looking that closely. Switched this one to a TODO.